common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -72,10 +72,6 @@ def build_xml(self, **kwargs):
             script_element.set("type", "loncapa/python")
             script_element.text = str(script)
 
-        # The problem has a child <p> with question text
```

Comment:
Moved question text into proper location-- within response type and using a label.